packages/common/src/ee/AiAgent/schemas/tools/toolRunQueryArgs.ts

```diff
@@ -122,6 +137,8 @@ Configuration Tips:
 - Specify exploreName, dimensions (for grouping/x-axis), and metrics (for y-axis values)
 - First dimension is the x-axis; additional dimensions can be used for series breakdown via groupBy
 - At least one metric is required for all chart types except table
+- chartConfig.xAxisDimension: Select the primary dimension from queryConfig.dimensions (typically dimensions[0]) useful for visualizing the data
+- chartConfig.yAxisMetrics: Select the metrics to display from queryConfig.metrics or tableCalculations useful for visualizing the data
```

Comment:
```suggestion
- chartConfig.xAxisDimension: Select the primary dimension from queryConfig.dimensions (typically dimensions[0])
- chartConfig.yAxisMetrics: Select the metrics to display from queryConfig.metrics or tableCalculations
```

the last bit seems unnecessary, unless it improves output?